packages/app/src/types/history/index.ts

```diff
@@ -7,12 +7,12 @@ export enum OperationType {
   DOWNLOAD_BACKUP = "DOWNLOAD_BACKUP",
   UPLOAD_BACKUP = "UPLOAD_BACKUP",
   RESET_PASSWORD = "RESET_PASSWORD",
-  ADD_VERIFIABLE_CREDENTIAL = "ADD_VERIFIABLE_CREDENTIAL",
+  NEW_VERIFIABLE_CREDENTIAL = "NEW_VERIFIABLE_CREDENTIAL",
```

Comment:
```suggestion
  NEW_VC = "NEW_VC",
```

Reply:
I think all interfaces between services (RPC actions, Event names, Paths, etc.) should use the full name `VERIFIABLE_CREDENTIAL` or `VERIFIABLE_PRESENTATION`. It makes the code easier to reason about and there is no path name to make the abbreviations clear.